ntoskrnl/include/internal/amd64/ke.h

```diff
@@ -420,6 +420,9 @@ VOID KiGetCacheInformation(VOID);
 VOID KiSetProcessorType(VOID);
 ULONG KiGetFeatureBits(VOID);
 VOID KiInitializeCpuFeatures(VOID);
+#if DBG
+VOID KiReportCpuFeatures(IN PKPRCB Prcb);
```

Comment:
```suggestion
VOID NTAPI KiReportCpuFeatures(_In_ PKPRCB Prcb);
```

Reply:
AMD64 functions in `cpu.c` do not have that, so I'm following existing conventions.

Reply:
Just following other function declarations in `ke.h` here.

Reply:
For what it's worth, `KiSetTrapContext` has `_In_`.

While here, `KiGetUserModeStackAddress` could have `s/void/VOID/`.